### DISCUSSION TITLE
Unresolved issues with #9068

### DISCUSSION THREAD

**Discussion by suzukimain** [huggingface/diffusers#9404] [Q&A] [answered]
Hello.
I am trying to solve a [problem](https://github.com/huggingface/diffusers/pull/9068#issuecomment-2266657636) that I could not solve when I did Pull requests in #9068 before, can you please advise me about the message that is displayed when reducing the filtering strength of the safety checker?

https://github.com/huggingface/diffusers/blob/c1a96a0e590cf0bc3d96c6cb4b90113d63f53ec8/src/diffusers/pipelines/stable_diffusion/safety_checker.py#L72-L79

* I am currently thinking of fixing it as follows.

```python
logger.warning(
    f"You have weakened the filtering strength of safety checker. Ensure"
    " that you abide to the conditions of the Stable Diffusion license and do not expose unfiltered"
    " results in services or applications open to the public. Both the diffusers team and Hugging Face"
    " strongly recommend to keep the safety filter enabled in all public facing circumstances, disabling"
    " it only for use-cases that involve analyzing network behavior or auditing its results. For more"
    " When reducing the filtering strength, take the same action as when disabling the safety checker."
    " information, please have a look at https://github.com/huggingface/diffusers/pull/254 ."             
)
```

**Comment by asomoza** [comment]
cc: @stevhliu

**Comment by stevhliu** [comment]
Hi there, I took a quick look at your PR. Some questions I have:

- Would this message only be displayed if the safety checker level is < `0.0`?
- Is there still a message that gets displayed when the safety checker is **disabled** instead of **changing** its strength level?

**Reply by suzukimain** [reply]
hi, @stevhliu Thanks for the response.
1. yes. This message is displayed only when the safety checker level is less than 0.0
2. yes, It is left.

**Reply by stevhliu** [reply, marked as answer]
Cool, feel free to add it to your PR then!

**Reply by suzukimain** [reply]
Thank you!
